manifests/12_prometheus_rule.yaml

```diff
@@ -8,11 +8,59 @@ metadata:
     role: alert-rules
 spec:
   groups:
-    - name: marketplace.rules
+    - name: marketplace.community_operators.rules
       rules:
-        - alert: community-operators-alert
-          expr: increase(app_registry_request_total{code!="200",opsrc="community-operators"}[10m]) > 10
-        - alert: certified-operators-alert
-          expr: increase(app_registry_request_total{code!="200",opsrc="certified-operators"}[10m]) > 10
-        - alert: redhat-operators-alert
-          expr: increase(app_registry_request_total{code!="200",opsrc="redhat-operators"}[10m]) > 10
+        - record: app_registry:community_operators:1xx_response
+          expr: sum(app_registry_request_total{code=~"1..",opsrc="community-operators"})
+        - record: app_registry:community_operators:2xx_response
+          expr: sum(app_registry_request_total{code=~"2..",opsrc="community-operators"})
+        - record: app_registry:community_operators:3xx_response
+          expr: sum(app_registry_request_total{code=~"3..",opsrc="community-operators"})
+        - record: app_registry:community_operators:4xx_response
+          expr: sum(app_registry_request_total{code=~"4..",opsrc="community-operators"})
+        - record: app_registry:community_operators:5xx_response
+          expr: sum(app_registry_request_total{code=~"5..",opsrc="community-operators"})
+        - alert: community_operators_alert
```

Comment:
Convention is that the alert name should be CamelCase (and the `_alert` suffix it not necessary, as we know it's an alert :slightly_smiling_face: ), and try to be more descriptive in what this alert means. Something like `CommunityOperatorConnectionErrors`.